static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-sign-in-cta.js

```diff
@@ -56,5 +56,5 @@ export const acquisitionsBannerSignInCtaTemplate = (
             class="u-faux-block-link__overlay"
             target="_blank"
             href="${params.linkUrl}"
-        />
```

Comment:
ah, is this a fix for the issue we saw earlier where all the links on the page were pointing to the support site (including links to privacy policy etc)?

Reply:
yes, believe so - in the instance where both banners are shown. testing the changes now